eyeshade/migrations/0001_transactions/up.sql

```diff
@@ -2,8 +2,8 @@ select execute($$
 
 insert into migrations (id, description) values ('0001', 'transactions');
 
-create type account_type as enum ('channel', 'owner', 'uphold', 'internal');
```

Comment:
these should be added in a new migrations file `eyeshade/migrations/0002_ads/up.sql` which modifies the existing enums